minima.gemspec

```diff
@@ -19,20 +19,6 @@ Gem::Specification.new do |spec|
   spec.bindir        = "exe"
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
 
-  spec.post_install_message = <<-msg
-
-----------------------------------------------
-Thank you for installing minima 2.0!
-
-Minima 2.0 comes with a breaking change that
-renders '<your-site>/css/main.scss' redundant.
-That file is now bundled with this gem as
-'<minima>/assets/main.scss'.
-
-More Information:
-https://github.com/jekyll/minima#customization
-----------------------------------------------
-
 msg
 
```

Comment:
you missed a couple of lines.. its L#22 - L#37 :smiley: 
[Reference](https://github.com/jekyll/minima/commit/8be22dc988b39ad846d493b7a52115acac2a7d50)